compiler/damlc/tests/daml-test-files/EvaluationOrder.daml

```diff
@@ -0,0 +1,8 @@
+-- @ERROR Aborted: BOOM
```

Comment:
Don't we need a copyright header here?

Reply:
:confused: CI doesn’t seem to think so?

Reply:
Added one anyway :shrug:

Reply:
There's a `NO_AUTO_COPYRIGHT` file in that folder. Given the number of files in there I suppose we should try to make that more granular at some point.

Reply:
It doesn’t matter much. They are all just test files so I don’t think having the copyright header there is particularly important. I would treat them similar to our templates.